assets/charts/components/cert-manager/templates/cainjector-psp.yaml

```diff
@@ -12,7 +12,8 @@ metadata:
     app.kubernetes.io/component: "cainjector"
     helm.sh/chart: {{ include "cainjector.chart" . }}
   annotations:
-    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+    # XXX: Lokomotive specific change
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default,runtime/default'
```

Comment:
Is there an upstream issue/PR opened with those changes?

Reply:
Yes https://github.com/jetstack/cert-manager/issues/3374

Reply:
Cool, thanks!